x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -196,12 +184,23 @@ export const AllCasesList = React.memo<AllCasesListProps>(
 
     return (
       <>
-        <ProgressLoader
+        <EuiProgress
           size="xs"
           color="accent"
           className="essentialAnimation"
-          $isShow={isLoading || isLoadingCases || isLoadingColumns}
+          css={
+            isLoading || isLoadingCases || isLoadingColumns
+              ? css`
```

Comment:
There's a pattern emerging where styles like this are actually defined in a `[component].styles.ts` file.  It makes the divide easier to understand, and makes the code in the component cleaner.

Reply:
This is a very good suggestion @clintandrewhall 
I discussed with the team and we agreed to move all styles to `[component].styles.ts` file.
Is that okay if I handle it in a separate PR?